packages/dsi-logo-maker/functions/logoAnimatedSVG/index.js

```diff
@@ -7,7 +7,7 @@ import {
   getIndexModule,
 } from "../../utils/blocks";
 import { Unit } from "../../utils/blocks-components";
-import { useDrawLoop } from "../../utils/drawLoopHook";
+import { useDrawLoop } from "../../utils/hooks";
```

Comment:
I think the font loading is missing in this function :blush:

Reply:
Wooops!